packages/entity-server/src/routes/hierarchy/relations/RelationsResponseBuilder.ts

```diff
@@ -128,7 +129,7 @@ export class RelationsResponseBuilder {
     const { entity, hierarchyId, allowedCountries } = this.ctx;
     const { type: descendantType, filter } = this.ctx.descendant;
 
-    const descendants = await entity.getDescendants(hierarchyId, {
+    const descendants = await entity.getRelatives(hierarchyId, {
```

Comment:
So I guess it would now be possible for someone to request e.g. country descendants and facility ancestors, and it would respond with something pseudo-sensible? That's probably fine to leave on the end user to get right, just noting it as a bit of a logical weirdness